### PR TITLE
Support to MicrosoftEdge on chromium

### PR DESCRIPTION
**Pull request by v1-wizard**
### Description
This PR adds support to MicrosoftEdge containerization. 

### Motivation and Context
* https://github.com/SeleniumHQ/docker-selenium/issues/936
* https://github.com/SeleniumHQ/docker-selenium/issues/350
* I want to use `MicrosoftEdge` with `testcontainers` in my project for testing.

### Types of changes
- [x] New feature (non-breaking change which adds functionality)

### Checklist
- [x] I have read the [contributing](https://selenium.dev/documentation/en/contributing/) document.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Review by diemol — COMMENTED** [review]
Thanks for this, @v1-wizard!
I haven't reviewed everything, but it looks good!

Seems that the commits are not associated with your GitHub user, could you please figure that out, sign the CLA, and then we can start working on merging this?

**Comment by diemol** [comment]
Please disregard my previous comment, I see you already fixed it.

**Comment by v1-wizard** [comment]
@diemol hi, nice to meet you. I see that one of the test is failed, but actually `make test_video` work perfectly on my laptop (sounds, like tipical programmer joke, lol). Could you help me with it?

**Comment by diemol** [comment]
Sure, let me have a look and see what is going on. I probably need a couple of days to finish something first.

**Comment by v1-wizard** [comment]
@diemol Hi, I found and fixed whole problem with tests. =)

**Comment by diemol** [comment]
> @diemol Hi, I found and fixed whole problem with tests. =)

😆 I just saw the commit that made the tests pass, sometimes it is the little things!

Thank you for taking the time and making all tests pass! I will merge this soon.

File: README.md [hunk 1]
```diff
@@ -242,6 +265,7 @@ configs = [
     "selenium/standalone-firefox:4.0.0-beta-1-prerelease-20210207", "{\"browserName\": \"firefox\"}",
     "selenium/standalone-chrome:4.0.0-beta-1-prerelease-20210207", "{\"browserName\": \"chrome\"}",
     "selenium/standalone-opera:4.0.0-beta-1-prerelease-20210207", "{\"browserName\": \"operablink\"}"
+    "selenium/standalone-edge:4.0.0-beta-1-prerelease-20210207", "{\"browserName\": \"msedge\"}"
```
Comment (diemol) [inline README.md]:
```suggestion
    "selenium/standalone-opera:4.0.0-beta-1-prerelease-20210207", "{\"browserName\": \"operablink\"}",
    "selenium/standalone-edge:4.0.0-beta-1-prerelease-20210207", "{\"browserName\": \"msedge\"}"
```

File: README.md [hunk 2]
```diff
@@ -73,6 +73,11 @@ $ docker run -d -p 4444:4444 -v /dev/shm:/dev/shm selenium/standalone-chrome:4.0
 $ docker run -d -p 4444:4444 -v /dev/shm:/dev/shm selenium/standalone-opera:4.0.0-beta-1-prerelease-20210207
 ```
 
+![Edge](https://raw.githubusercontent.com/alrra/browser-logos/main/src/edge/edge_24x24.png) Microsoft Edge
+``` bash
+$ docker run -d -p 4444:4444 -v /dev/shm:/dev/shm selenium/standalone-edge:4.0.0-beta-1-prerelease-20210201
```
Comment (diemol) [inline README.md]:
```suggestion
$ docker run -d -p 4444:4444 -v /dev/shm:/dev/shm selenium/standalone-edge:4.0.0-beta-1-prerelease-20210207
```

**Review by diemol — APPROVED** [review]
Thank you, @v1-wizard!

**Comment by diemol** [comment]
I need to correct some dates, and then I will release new images tomorrow.

**Pull request merged**